jobs/types/audit_errors.py

```diff
@@ -222,3 +222,13 @@ def __init__(self, id_property, model_id, target_kind, target_id):
             'but no such model exists' % (
                 id_property, python_utils.convert_to_bytes(target_id),
                 target_kind))
+
+
+class ModelCanonicalNameMismatchError(BaseAuditError):
+    """Error class for models that have mismatching names."""
+
+    def __init__(self, model):
+        super(ModelCanonicalNameMismatchError, self).__init__(model)
+        self.message = (
+            'Entity name %s in lowercase does not match '
+            'canonical name %s' % (model.name, model.canonical_name))
```

Comment:
Should we add _topic_errors.py_ file for this?

Reply:
Or _topic_audit_errors.py_

Reply:
Then in that case should we rename the current audit _errors to base_model_audit_errors?

Reply:
@rohitkatlaa  @vojtechjelinek  shouldn't we then refactor the current audit_errors for user_audit_errors as well?

Reply:
Do let me know I can create a separate PR with the refactoring of audit_errors.

Reply:
Yes, makes sense, please create an issue for this so that it is tracked.